tools/keytools/wolfBootSignTool.vcxproj

```diff
@@ -172,7 +175,13 @@
     <ClCompile Include="..\..\lib\wolfssl\wolfcrypt\src\tfm.c" />
     <ClCompile Include="..\..\lib\wolfssl\wolfcrypt\src\wc_port.c" />
     <ClCompile Include="..\..\lib\wolfssl\wolfcrypt\src\wolfmath.c" />
-    <ClCompile Include="sign.c" />
+    <ClCompile Include="..\..\src\delta.c" />
+    <ClCompile Include="sign.c">
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">..\..\lib\wolfssl;..\..\include;..\..\include;.;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">..\..\lib\wolfssl;..\..\include;.;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Can you please replace the forward slashes with backslashes? Or just use my project file changes in #161.